### DISCUSSION TITLE
Qual é o link da EPG da Rakuten Family?

### DISCUSSION THREAD

**Discussion by thomraider12** [inspirationlinks/lista-tuga#8] [Q&A]
Gostava de saber qual o link que contém os dados de EPG do canal Rakuten Family. São alguns links, e deve saber qual é.

Obrigado.

**Comment by iprtl** [comment]
Ola, por acaso nao sei, ainda nao encontrei quem inclua, se soubesse ja estava na lista

**Reply by thomraider12** [reply]
Sem problema
